spinel/util.py

```diff
@@ -19,15 +19,8 @@
 import binascii
 import sys
 
-def hexify_chr(s):
-    if isinstance(s, str) and sys.version_info[0] == 2:
-        s = ord(s)
-    return "%02X" % s
-
-def hexify_int(i): return "%02X" % i
-def hexify_bytes(data): return str(list(map(hexify_chr,data)))
 def hexify_str(s,delim=':'):
-    hex_str = binascii.b2a_hex(bytearray(s, "utf-8")).decode("utf-8")
+    hex_str = binascii.hexlify(s.decode('utf-8'))
```

Comment:
This line causes a traceback: `AttributeError: 'str' object has no attribute 'decode'`
I suggest changing it to `hex_str = binascii.hexlify(s.encode('utf-8')).decode('utf-8')`

Reply:
how to reproduce this?

Reply:
Try:
```
debug 1
diag start
```

Reply:
thank you, updated!